assemblyline_core/submission_client.py

```diff
@@ -232,7 +240,7 @@ def _ready_file(self, local_path: str, expiry, classification) -> Tuple[str, int
                 local_path = extracted_path
 
             self.datastore.save_or_freshen_file(fileinfo['sha256'], fileinfo, expiry,
-                                                al_meta['classification'], redis=self.redis)
```

Comment:
Why did we stop passing the redis connection?

Reply:
it doesn't use it any more since we switched from redis locks to elastic versioning